jobs/Backend/Task/ExchangeRateApi/Exceptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Net;
+
+public class CurrencyNotFoundException : Exception
```

Comment:
👍 Nice to use a custom exception, I haven't think about that 😅 (fellow candidate here)

Reply:
You are welcome to grab this idea :)